modin/pandas/indexing.py

```diff
@@ -296,6 +338,10 @@ def __getitem__(self, row_lookup, col_lookup, ndim):
             The global row index to retrieve data from.
         col_lookup : slice(None), range or np.ndarray
             The global col index to retrieve data from.
+        row_multiindex_full_lookup : bool
+        col_multiindex_full_lookup : bool
+        row_scalar : bool
+        col_scalar : bool
```

Comment:
some nitpicking - ideally this docstring should explain what parameters are here for, what they do etc.

Reply:
updated